src/Azure.Functions.Cli/Common/DurableManager.cs

```diff
@@ -65,12 +65,18 @@ private void SetConnectionStringAndTaskHubName()
                     {
                         // If the version is (explicitly) 2.0, prepend path to 'durableTask' with 'extensions'
                         _connectionStringKey = hostSettings?.extensions?.durableTask?.AzureStorageConnectionStringName ?? _connectionStringKey;
+
+                        // Allow uppercase or lowercase hub name in host.json
                         _taskHubName = hostSettings?.extensions?.durableTask?.HubName ?? _taskHubName;
+                        _taskHubName = hostSettings?.extensions?.durableTask?.hubName ?? _taskHubName;
                     }
                     else
                     {
                         _connectionStringKey = hostSettings?.durableTask?.AzureStorageConnectionStringName ?? _connectionStringKey;
+
+                        // Allow uppercase or lowercase hub name in host.json
                         _taskHubName = hostSettings?.durableTask?.HubName ?? _taskHubName;
+                        _taskHubName = hostSettings?.durableTask?.hubName ?? _taskHubName;
```

Comment:
That's an interesting way to solve it. I think it's fine, but a little bit hard to understand. An alternative approach might be to use `JObject.GetValue(string, StringComparison)` as described here: https://stackoverflow.com/a/19205083/2069

Reply:
how does the runtime/durable extension read that value?

Reply:
The runtime reads it in as a `JObject` and populates a custom .NET type defined by the extension. I don't remember the exact details, but it does this in a way that is case-insensitive.

Reply:
That's right, I remember [this](https://github.com/Azure/azure-functions-host/blob/fd6cde34dfa103949d9fb003915969528eaf810c/src/WebJobs.Script.WebHost/Management/WebFunctionsManager.cs#L289-L318)